docs/versionning.md

```diff
@@ -6,42 +6,33 @@ UData uses [semantic versionning][semver] to publish its published releases.
 
 ### Branches management
 
-There is two main branches on the [UData git repository][github]: `master` and `dev`
+There is a main branch on the [UData git repository][github], `master`,
+and some maintenance branches `vX.Y`
 
 The `master` is the stable development branch on which:
 
-- bug fixes should occurs (unless is only present on `dev` branch)
-- bugfix or security upgrades (only) are done
+- bug fixes should occurs (unless is only present on a maintenance branch)
+- security upgrades are done (unless is only necessary on a maintenance branch)
+- the incoming new features (and their bug fixes)
+- the refactoring
+- the dependencies upgrades
 - translations are done
 - releases are done
 
-The `dev` branch hosts:
+The `vX.Y` maintenance branches host:
 
-- the incoming new features (and their bug fixes)
-- the factoring
-- the dependencies upgrades
-- the bug fixes associated to these
+- the bug fixes and security upgrades associated to theses versions only
+- the backported bug fixes and security upgrades
 
-There is always at least one minor version between the `dev` and the `master branch`
-(*ie.* `1.0.2.dev` on `master` and `1.1.0.dev` on `dev`).
+Every minor version has a maintenance branch `vX.Y` and every version has a git tag `vX.Y.Z`.
 
-
-Every version has a git tag `X.Y.Z` and then when bug fixes need to be applied
-without publishing new changes, there will be a version branch `X.Y`
+Otherwise the version is `X.Y.Z.dev` (1.1.7.dev)
```

Comment:
If we follow semver, should not it be `1.1.7-dev` instead? Cf. [hyphen separated suffix](http://semver.org/#spec-item-9).

Reply:
It follows [Python versionning, developmental releases](https://www.python.org/dev/peps/pep-0440/#developmental-releases)